src/mumble/Settings.cpp

```diff
@@ -759,6 +759,7 @@ void Settings::load(QSettings* settings_ptr) {
 	SAVELOAD(bQoS, "net/qos");
 	SAVELOAD(bReconnect, "net/reconnect");
 	SAVELOAD(bAutoConnect, "net/autoconnect");
+	SAVELOAD(bDisablePublicList, "net/disablepubliclist");
```

Comment:
I think we should keep this option in the `ui` section in the settings file in order to keep old settings working (some might have set this option manually before) :point_up:

Reply:
Thought about that as well and figured if that is the case someone will call me out on this. Will change it